debian/conf/distributions

```diff
@@ -140,4 +140,12 @@ Codename: trixie
 Architectures: amd64 i386 source
 Components: main prerelease
 SignWith: F473DD4473365DE1
-Tracking: minimal
\ No newline at end of file
+Tracking: minimal
+
+# mantic (Ubuntu 23.10) reaches EOL in July 2024
+Origin: matrix.org
+Codename: mantic
+Architectures: amd64 i386 source
+Components: main prerelease
+SignWith: F473DD4473365DE1
+Tracking: minimal
```

Comment:
I haven't done this before, but I'm guessing from previous changes to this file that we use the same SignWith key for each ubuntu version.